Sources/UI/ActionButton/ActionButton.swift

```diff
@@ -8,22 +8,13 @@
 
 import SwiftUI
 
-public class ActionButtonViewModel: ObservableObject {
-  @Published public var title: String = ""
-  @Published public var font: Font = .system(size: 18)
-  @Published public var textColor: Color = .white
-  @Published public var cornerRadius: ActionButton.CornerRadiusType = .custom(0)
-  @Published public var backgroundType = ActionButton.Background.plain(.blue)
-  @Published public var borderColor: Color = .clear
-  @Published public var borderWidth: CGFloat = 0
-  @Published public var extraImage: ActionButton.ExtraImage?
```

Comment:
Any particular reason why we need an `ObservableObject` for the View's properties?

Reply:
Ahh, I see! Interesting. What triggered me to ask this question was the builder pattern that you used, from previous experience usually you create a copy of self, modify the intended property, then return it, whereas here it was through a `ObservableObject`. 

Thanks for explaining!